### PR TITLE
feat: Practice tab home skeleton

### PR DESCRIPTION
**Pull request by Adaendra**
### Related PR
- https://github.com/RoadTripMoustache/kana_to_kanji/pull/259
- https://github.com/RoadTripMoustache/kana_to_kanji/pull/257

## 📖 Description
Add practice skeleton page.

### ⁉️ Related Issues
closes: #162 

### 🖼️ Screenshots:

<details>
    <summary>The skeleton page</summary> 

image

</details>

### 🧪 How to test the change?
- Open the application
- Click on the "Practice" tab

### ☑️ Checklist before requesting a review
- [x] I have performed a self-review of my code.
- [x] I have added/updated tests.
- [x] Make sure to add either one of the following labels: `version: Major`,`version: Minor` or `version: Patch`.

### CODE REVIEW

File: lib/src/practice/quiz/practice_view.dart [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  Widget build(BuildContext context) =>
+      ViewModelBuilder<PracticeViewModel>.reactive(
+          viewModelBuilder: () => PracticeViewModel(GoRouter.of(context)),
+          builder: (context, viewModel, child) => AppScaffold(
+              resizeToAvoidBottomInset: true,
+              showBottomBar: true,
+              appBar: AppBar(
+                title: SizedBox(
+                  width: double.infinity,
+                  child: FilledButton(
```
Comment (apomalyn) [inline lib/src/practice/quiz/practice_view.dart:22]: The button should be on the `body` and the `appBar` left empty

**Review by apomalyn — CHANGES_REQUESTED** [review]

File: lib/src/practice/quiz/practice_view.dart [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+                    onPressed: () {},
+                    child: const Text("Practice"),
```
Comment (apomalyn) [inline lib/src/practice/quiz/practice_view.dart:24]: Can you localize this one

**Comment by apomalyn** [comment]
Can you update [SplashViewModel l.21](https://github.com/RoadTripMoustache/kana_to_kanji/blob/f8ad8b84e3594c65681d2680ac4c9f0827598fc7/lib/src/splash/splash_view_model.dart#L21) to redirect to this new view instead of `PrepareQuizView`

File: lib/src/practice/quiz/practice_view.dart (hunk 1 above)
Comment (Adaendra) [reply lib/src/practice/quiz/practice_view.dart]: Updated

File: lib/src/practice/quiz/practice_view.dart (hunk 2 above)
Comment (Adaendra) [reply lib/src/practice/quiz/practice_view.dart]: Updated

**Review by apomalyn — APPROVED** [review]

File: lib/src/core/services/api_service.dart
Comment (apomalyn) [inline lib/src/core/services/api_service.dart:1]: Not needed 😂 ça devient un running gag

**Review by apomalyn — CHANGES_REQUESTED** [review]

File: lib/src/core/services/api_service.dart
Comment (Adaendra) [reply lib/src/core/services/api_service.dart:1]: Quand on est fatigué, on l'oublie souvent xD

**Review by apomalyn — APPROVED** [review]

**Pull request merged**